QWeb/internal/screenshot.py

```diff
@@ -272,3 +283,56 @@ def get_url():
 def get_source():
     driver = browser.get_current_browser()
     return driver.page_source
+
+
+def chromium_full_screenshot(driver, filepath):
+
+    def send(cmd, params):
+        resource = f"/session/{driver.session_id}/chromium/send_command_and_get_result"
+
+        # pylint:disable=W0212
+        url = driver.command_executor._url + resource
+        body = json.dumps({'cmd': cmd, 'params': params})
+        response = driver.command_executor._request('POST', url, body)
+        return response.get('value')
+
+    def evaluate(script):
+        response = send('Runtime.evaluate', {
+            'returnByValue': True,
+            'expression': script
+        })
+        return response['result']['value']
+
+    metrics = evaluate("({"
+                       "width: Math.max(window.innerWidth, "
+                       "document.body.scrollWidth, "
+                       "document.documentElement.scrollWidth)|0,"
+                       "height: Math.max(innerHeight, document.body.scrollHeight, "
```

Comment:
Should `innerHeight` be `window.innerHeight`?

Reply:
Thanks, yes I think it should. Will resolve in a new commit.